staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -128,7 +128,10 @@ func (r *rudimentaryErrorBackoff) OnError(error) {
 	r.lastErrorTimeLock.Lock()
 	defer r.lastErrorTimeLock.Unlock()
 	d := time.Since(r.lastErrorTime)
-	if d < r.minPeriod {
+	if d < 0 {
+		// reset when time hopping
```

Comment:
No one will understand this-- how about "If the time moves backwards for any reason, do nothing"

Reply:
I hear rumors that go 1.9 fixes this, so please investigate and add a "// TODO: remove this after go 1.8 is no longer supported" line if that is true.

Reply:
makes sense to me